torch/testing/_internal/common_methods_invocations.py

```diff
@@ -420,6 +420,42 @@ def sample_inputs_linalg_norm(op_info, device, dtype, requires_grad):
                             dim=(0, 1))))
         return inputs
 
+def sample_inputs_linalg_vector_norm(op_info, device, dtype, requires_grad):
+    test_sizes = [
+        (S,),
+        # For test_variant_consistency_jit, if torch.half inputs are too
+        # large, double gradient calculations end up containing NaN values
+        # because multiplications overflow to infinity. So we need to use
+        # a very small input size
+        (2, 2) if dtype == torch.half else (S, S),
+    ]
+    ords = (None, 0, 0.5, 1, 2, 3.5, inf, -3.5, -inf)
+    inputs = []
+
+    def get_all_unique_dim_args(ndim):
```

Comment:
For simplicity let's write the test cases as tuples (shape, args) and just handwrite some interesting set of dims to inspect instead of programmatically generating all of them. There is only a 1D and 2D tensor, anyway, so the interesting arguments to dim are few.